api/admin/service.go

```diff
@@ -8,6 +8,8 @@ import (
 	"io/ioutil"
 	"net/http"
 
+	"github.com/ava-labs/avalanchego/ids"
```

Comment:
Super nit picky here, but can this import get moved to be with the other unnamed "github.com/ava-labs/avalanchego/..." imports?

Reply:
Did not notice that one tbh 😄 . Don't know why goimports did not do it..